packages/discovery-react-components/src/components/DocumentPreview/components/SimpleDocument/__tests__/SimpleDocument.test.tsx

```diff
@@ -135,4 +135,33 @@ describe('SimpleDocument', () => {
 
     getByText(passageText);
   });
+
+  it('renders a fallback component when nothing to display in document', () => {
+    const jsonDoc = {
+      document_id: '1234567890',
+      extracted_metadata: {
+        filename: 'i_am_a_file',
+        file_type: 'json'
+      },
+      result_metadata: {
+        collection_id: '1234'
+      }
+    };
+
+    let getByText: BoundFunction<GetByText> = noop;
+    act(() => {
```

Comment:
Is `act()` actually needed here?

Reply:
Not necessary. I remove it to simplify the test case.